client/tink_keyset_helper_test.go

```diff
@@ -0,0 +1,20 @@
+package client
+
+import (
+	"sort"
+	"strings"
+	"testing"
+)
+
+func TestNameOfSupportedTinkKeyTemplates(t *testing.T) {
+	supportedTemplates := make([]string, 0, len(tinkKeyTemplates))
```

Comment:
I recommend calling the function you're testing here instead of copying over the contents of that function.

Reply:
In my understanding, should I call the func "NameOfSupportedTinkKeyTemplates" here, and hard code the expected result as a string. Then, compare this string and the result from "NameOfSupportedTinkKeyTemplates" to decide whether this test is passed or not? Thanks!

Reply:
Yes that's right. You can see more examples of idiomatic testing style here: https://gobyexample.com/testing